src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
```

Comment:
Please do not commit files not require by the task

Reply:
not fixed